pkg/proxy/ipvs/proxier.go

```diff
@@ -1141,36 +1184,49 @@ func (proxier *Proxier) syncProxyRules() {
 			"-m", "set", "--match-set", proxier.lbMasqSet.Name,
 			"dst,dst",
 		)
-		writeLine(proxier.natRules, append(args, "-j", string(KubeMarkMasqChain))...)
-	}
-	if !proxier.lbWhiteListCIDRSet.isEmpty() || !proxier.lbWhiteListIPSet.isEmpty() {
-		// link kube-services chain -> kube-fire-wall chain
-		args := []string{"-m", "set", "--match-set", proxier.lbIngressSet.Name, "dst,dst", "-j", string(KubeFireWallChain)}
-		if _, err := proxier.iptables.EnsureRule(utiliptables.Append, utiliptables.TableNAT, kubeServicesChain, args...); err != nil {
-			glog.Errorf("Failed to ensure that ipset %s chain %s jumps to %s: %v", proxier.lbIngressSet.Name, kubeServicesChain, KubeFireWallChain, err)
-		}
-		if !proxier.lbWhiteListCIDRSet.isEmpty() {
+		writeLine(proxier.natRules, append(args, "-j", string(KubeFireWallChain))...)
+		// TODO: is not masq for esipp, then maybe droped for traffic from outside.
```

Comment:
It's hard to understand what this TODO is saying. Can you make it clearer or remove it?